packages/ember-toucan-form/src/-private/textarea-field.hbs

```diff
@@ -0,0 +1,13 @@
+<@form.Field @name={{@name}} as |field|>
+  <Form::Fields::Textarea
+    @label={{@label}}
+    @hint={{@hint}}
+    @error={{this.mapErrors field.rawErrors}}
+    @value={{field.value}}
+    @onChange={{field.setValue}}
+    @isDisabled={{@isDisabled}}
+    @rootTestSelector={{@rootTestSelector}}
+    name={{@name}}
+    ...attributes
+  />
```

Comment:
I'm getting the following lint error:

```bash
@crowdstrike/ember-toucan-form:lint: [lint:types] src/-private/textarea-field.hbs:2:3 - error TS7053: Unknown name 'Form::Fields::Textarea'. If this isn't a typo, you may be missing a registry entry for this value; see the Template Registry page in the Glint documentation for more details.
@crowdstrike/ember-toucan-form:lint: [lint:types]   Element implicitly has an 'any' type because expression of type '"Form::Fields::Textarea"' can't be used to index type 'Globals'.
@crowdstrike/ember-toucan-form:lint: [lint:types]     Property 'Form::Fields::Textarea' does not exist on type 'Globals'.
@crowdstrike/ember-toucan-form:lint: [lint:types] 
@crowdstrike/ember-toucan-form:lint: [lint:types]   2   <Form::Fields::Textarea
@crowdstrike/ember-toucan-form:lint: [lint:types]       ~~~~~~~~~~~~~~~~~~~~~~~
@crowdstrike/ember-toucan-form:lint: [lint:types]   3     @label={{@label}}
@crowdstrike/ember-toucan-form:lint: [lint:types]     ~~~~~~~~~~~~~~~~~~~~~
@crowdstrike/ember-toucan-form:lint: [lint:types] ... 
@crowdstrike/ember-toucan-form:lint: [lint:types]  11     ...attributes
@crowdstrike/ember-toucan-form:lint: [lint:types]     ~~~~~~~~~~~~~~~~~
@crowdstrike/ember-toucan-form:lint: [lint:types]  12   />
@crowdstrike/ember-toucan-form:lint: [lint:types]     ~~~~
```

Which is weird because I'm extending the unpublished registry with Toucan Core's registry at https://github.com/CrowdStrike/ember-toucan-core/pull/129/files#diff-84c037d42edbe42211468688395a6b1a14a5bbaee06c5b48b6125cfe93c48255R31

---

I'm also getting this:

```bash
[lint:types] ../../node_modules/.pnpm/ember-headless-form@1.0.0-beta.1_@glimmer+componen_ym5tcvhcc62nd3g4tiyqwpfw6i/node_modules/ember-headless-form/dist/components/headless-form.d.ts:135:16 - error TS2307: Cannot find module '@ember/modifier' or its corresponding type declarations.
[lint:types] 
[lint:types] 135     on: import("@ember/modifier").OnModifier;
[lint:types]                    ~~~~~~~~~~~~~~~~~
[lint:types] 
```

Which I added `ember-modifier` as a dependency for as well.  Any ideas?